docs/local_development.md

```diff
@@ -83,6 +87,7 @@ yarn run dev:chrome # or dev:firefox, dev:opera
 1. Go to Chrome's [extension page](chrome://extensions).
 2. Enable `Developer mode`.
 3. Either click `Load unpacked extension...` and select `./extension/extension/chrome` or drag that folder into the page.
+4. Transfer eth from metamask to Quill if using `yarn start-hardhat` in `./contracts`. 
 
 ### Firefox
```

Comment:
Could you add the same point to the firefox instructions too?

e.g. `4. Transfer ETH from a funded account to Quill - e.g. an imported MetaMask account`